Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,10 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+const wholeNumberPart = Math.floor(num);
+const decimalPart = (num % 1).toFixed(4);
```

Comment:
@RihannaP 
I took a look at your code, and I learned that using the modulus operator is a better approach for extracting the decimal part of a number. Thanks  🙏

Reply:
This is my code for extracting the decimal part:


`const decimalPart = (num - wholeNumberPart).toFixed(4);
`